pkg/pdatatest/plogtest/logs_test.go

```diff
@@ -55,150 +55,84 @@ func TestCompareLogs(t *testing.T) {
 				IgnoreResourceLogsOrder(),
 			},
 			withoutOptions: multierr.Combine(
-				errors.New("ResourceLogs with attributes map[testKey1:one] expected at index 0, found at index 1"),
-				errors.New("ResourceLogs with attributes map[testKey2:one] expected at index 1, found at index 0"),
+				errors.New("resources are out of order: resource map[testKey1:one] expected at index 0, found at index 1"),
+				errors.New("resources are out of order: resource map[testKey2:one] expected at index 1, found at index 0"),
 			),
 			withOptions: nil,
 		},
 		{
-			name: "resource-instrumentation-library-extra",
-			withoutOptions: multierr.Combine(
-				errors.New("ResourceLogs with attributes map[] does not match expected"),
-				errors.New("number of scope logs does not match expected: 1, actual: 2"),
-			),
+			name:           "resource-instrumentation-library-extra",
+			withoutOptions: errors.New("resource map[]: number of scopes doesn't match expected: 1, actual: 2"),
 		},
 		{
-			name: "resource-instrumentation-library-missing",
-			withoutOptions: multierr.Combine(
-				errors.New("ResourceLogs with attributes map[] does not match expected"),
-				errors.New("number of scope logs does not match expected: 2, actual: 1"),
-			),
+			name:           "resource-instrumentation-library-missing",
+			withoutOptions: errors.New("resource map[]: number of scopes doesn't match expected: 2, actual: 1"),
 		},
 		{
 			name: "resource-instrumentation-library-name-mismatch",
 			withoutOptions: multierr.Combine(
-				errors.New("ResourceLogs with attributes map[] does not match expected"),
-				errors.New("missing ScopeLogs with scope name: one"),
-				errors.New("unexpected ScopeLogs with scope name: two"),
+				errors.New("resource map[]: missing expected scope: one"),
+				errors.New("resource map[]: unexpected scope: two"),
 			),
 		},
 		{
-			name: "resource-instrumentation-library-version-mismatch",
-			withoutOptions: multierr.Combine(
-				errors.New("ResourceLogs with attributes map[] does not match expected"),
-				errors.New(`ScopeLogs with scope name "one" do not match expected`),
-				errors.New("scope version does not match expected: 1.0, actual: 2.0"),
-			),
+			name:           "resource-instrumentation-library-version-mismatch",
+			withoutOptions: errors.New(`resource map[]: scope "one": version doesn't match expected: 1.0, actual: 2.0`),
 		},
 		{
-			name: "logrecords-missing",
-			withoutOptions: multierr.Combine(
-				errors.New("ResourceLogs with attributes map[type:one] does not match expected"),
-				errors.New(`ScopeLogs with scope name "" do not match expected`),
-				errors.New("number of log records does not match expected: 1, actual: 0"),
-			),
+			name:           "logrecords-missing",
+			withoutOptions: errors.New(`resource map[type:one]: scope "": number of log records doesn't match expected: 1, actual: 0`),
```

Comment:
```suggestion
			withoutOptions: errors.New(`resource "map[type:one]": scope "": number of log records doesn't match expected: 1, actual: 0`),
```

I wonder if we should quote the attribute maps. To the reader, it serve the same purpose as a name, which we are quoting for scopes and metrics.

Reply:
I'm ok with quotes but not sure it improves readability. Maybe for just for consistency. I added quotes to names because there are a lot of empty names especially in scopes that would result in confusing two spaces, but it's not the case for maps

Reply:
👍 Either way is fine with me.

Reply:
Ok, I applied quotes to all such identifiers consistently